src/Microsoft.Azure.NotificationHubs/RegistrationDescription.cs

```diff
@@ -53,7 +51,7 @@ public abstract class RegistrationDescription : EntityDescription
         /// Initializes a new instance of the <see cref="RegistrationDescription"/> class.
         /// </summary>
         /// <param name="registration">The registration.</param>
-        public RegistrationDescription(RegistrationDescription registration)
+        protected RegistrationDescription(RegistrationDescription registration)
```

Comment:
I agree constructors on abstract classes should be `protected`, but I'd rather not change this now. It doesn't hurt to have it `public`.

Reply:
Reverted